base/exports.jl

```diff
@@ -1180,8 +1180,10 @@ export
 # multiprocessing
     addprocs,
     asyncmap,
+    CachingPool,
     ClusterManager,
     default_worker_pool,
+    define,
```

Comment:
This is a pretty generic name to start exporting for this. Is it more like a `remoteset!` or `remotedefine` maybe?